src/components/taskbar/index.js

```diff
@@ -52,13 +52,13 @@ const Taskbar = () => {
   };
 
   const changebatterystatus = (bt) => {
-    let level = bt.level * 100 || 100;
+    let level = bt.level * 100;
```

Comment:
that `|| 100` was for browsers that don't return battery percentage, It takes 100 as default.